app/views/costs/CostsSummaryPage.scala.html

```diff
@@ -0,0 +1,157 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import uk.gov.hmrc.cataloguefrontend.connector.Team
+@import uk.gov.hmrc.cataloguefrontend.costs.ServiceAndEnvironment
+@import views.html.helper.select
+@import uk.gov.hmrc.cataloguefrontend.service.CostEstimateConfig
+
+@this(viewMessages: ViewMessages)
+
+@(
+        configs             : Seq[ServiceAndEnvironment],
```

Comment:
the naming could be clearer - `ServiceAndEnvironment` is called `configs` - and rather than representing a `ServiceName` and `Environment`. It seems to be (practically) `Map[ServiceName, Map[Environment, DeploymentSize]`. It might be better modelled as such (which is clear) - extractors being recognised provided ones (`get`) (and a couple of helpers if necessary).